### PR TITLE
Update NE road properties

### PR DESCRIPTION
**Pull request by zerebubuth**
Connects to #896.

@nvkelso could you review, please?

### CODE REVIEW

File: integration-test/896-ne-shield-enums-2.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    { 'source': 'naturalearthdata.com', 'kind': 'highway',
+      'level': None })
+assert_no_matching_feature(
+    7, 63, 42, 'roads',
+    { 'source': 'naturalearthdata.com', 'kind': 'highway',
+      'namealt': None })
+assert_no_matching_feature(
+    7, 63, 42, 'roads',
+    { 'source': 'naturalearthdata.com', 'kind': 'highway',
+      'namealtt': None })
```
Comment (rmarianski) [inline integration-test/896-ne-shield-enums-2.py]: Should these tests be for `None` or `type(None)`?

**Comment by nvkelso** [comment]
👍 (modulo Rob's comment above)

File: integration-test/896-ne-shield-enums-2.py (hunk 1 above)
Comment (zerebubuth) [inline integration-test/896-ne-shield-enums-2.py]: These are negative matches (`assert_no_matching_feature`), so the tests could be translated as "assert that there are no NE highways with any `level`, ans so forth. The aim is to make sure that all of the `level`, `namealt` and `namealtt` properties have been removed.

Comment (rmarianski) [inline integration-test/896-ne-shield-enums-2.py]: Ah ok, sorry I didn't catch that when reading through it initially. :+1:

**Pull request merged**